bot.py

```diff
@@ -437,27 +437,67 @@ def main_color(self):
         ctx = SimpleNamespace(bot=self, guild=self.modmail_guild)
         converter = commands.EmojiConverter()
 
-        blocked_emoji = self.config.get('blocked_emoji', '🚫')
         sent_emoji = self.config.get('sent_emoji', '✅')
+        blocked_emoji = self.config.get('blocked_emoji', '🚫')
 
-        if blocked_emoji not in UNICODE_EMOJI:
+        if sent_emoji not in UNICODE_EMOJI:
             try:
-                blocked_emoji = await converter.convert(
-                    ctx, blocked_emoji.strip(':')
+                sent_emoji = await converter.convert(
+                    ctx, sent_emoji.strip(':')
                 )
             except commands.BadArgument:
-                pass
+                logger.warning(info(f'Sent Emoji ({sent_emoji}) '
+                                    f'is not a valid emoji.'))
+                del self.config.cache['sent_emoji']
+                await self.config.update()
 
-        if sent_emoji not in UNICODE_EMOJI:
+        if blocked_emoji not in UNICODE_EMOJI:
             try:
-                sent_emoji = await converter.convert(
-                    ctx, sent_emoji.strip(':')
+                blocked_emoji = await converter.convert(
+                    ctx, blocked_emoji.strip(':')
                 )
             except commands.BadArgument:
-                pass
+                logger.warning(info(f'Blocked emoji ({blocked_emoji}) '
+                                    'is not a valid emoji.'))
+                del self.config.cache['blocked_emoji']
+                await self.config.update()
 
-        if str(message.author.id) in self.blocked_users:
+        account_age = self.config.get('account_age', 0)
+        try:
+            account_age = float(account_age)
+            if account_age < 0 or not account_age.is_integer():
+                raise ValueError
+        except ValueError:
+            logger.warning('The age limit needs to be a whole number, '
+                           f'not "{account_age}".')
+            del self.config.cache['account_age']
+            await self.config.update()
+        else:
+            account_age = int(account_age)
```

Comment:
why not leave it as a float?

Reply:
It's left as a float now.

Reply:
Now uses the international time standard ISO8601 duration format for `account_age`.